PointOfInteraction/SmartPointOfInteraction/doc/spec.md

```diff
@@ -0,0 +1,73 @@
+
+
+This entity is purely virtual, is not a device mapping.
+
+## Data Model
+
++ `id` : Unique identifier. 
+
++ `type` : Entity type. It must be equal to `SmartPointOfInteraction`.
+
++ `category` : Defines the type of interaction.
+    + Attribute type: List of [Text](http://schema.org/Text)
+    + Allowed values: `information`, `entertainment`, `infotainment`, `co-creation` or any other extended value defined by the application.
+    + Mandatory
+    
++ `areaCovered` : Defines the area covered by the Smart Point of Interaction using geoJSON format.
+    + Attribute type: `geo:json`.
+    + Normative References: [https://tools.ietf.org/html/rfc7946](https://tools.ietf.org/html/rfc7946)
+    + Optional    
+    
++ `applicationUrl` : This field specifies the real URL containing the solution or application (information, co-creation, etc) while the SmartSport 'announcedUrl' field specifies the broadcasted URL which could be this same URL or a shortened URL.
+    + Attribute type: [URL](https://schema.org/URL)
+    + Mandatory    
+
++ `availability`: Specifies the time intervals in which this interactive service is available, but this is a general information while SmartSpots have their own real availability in order to allow advanced configurations. The format is an structured value which must contain a subproperty per each required functionality interval, indicating when the functionality is active. If nothing specified (or null) it will mean that the functionality is always on. The syntax must be conformant with schema.org [openingHours specification](https://schema.org/openingHours). For instance, a service which is only active on dayweeks will be encoded as "availability": "Mo,Tu,We,Th,Fr,Sa 09:00-20:00". 
+    + Attribute type: [StructuredValue](https://schema.org/StructuredValue)
+    + Mandatory. It can be null.
+
++ `refRelatedEntity` : List of entities improved with this Smart Point of Interaction. The entity type could be any such as a “Parking”, “Point of Interest”, etc.
+    + Attribute type: List of entities.
+    + Optional    
+
++ `refSmartSpot` : References to the “Smart Spot” devices which are part of the Smart Point of Interaction.
+    + Attribute type: Reference to one or more entity of type [SmartSpot](https://github.com/Fiware/dataModels/blob/master/SmartPointOfInteraction/SmartSpot/doc/spec.md)
+    + Optional    
+
+## Examples of use
+
+```json
+{
+  "id": "SPOI-ES-4326",
+  "type": "SmartPointOfInteraction",
+  "category": ["co-creation"],
+  "areaCovered": {                           
+    "type": "Polygon",
+    "coordinates": [[
+      [25.774, -80.190],
+      [18.466, -66.118], 
+      [32.321, -64.757], 
+      [25.774, -80.190] 
+    ]]
+  },
+  "applicationUrl": "www.siidi.eu",
```

Comment:
As an URL, I would probably add the schema or protocol to the example, I mean: "https://ww.siidi.eu" :)